scripts/radix_git_status.sh

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
+
+set completion-ignore-case on
+set show-all-if-ambiguous on
+
+ERROR_MSG_NO_SUCH_FOLDER="___folder_does_not_exist___"
+
+RADIXDLT_JAVA_COMMON_FIRST_COMMIT_HASH='1b6b6c46c9a5b87ec29a32b47bf96e15bdfbf165'
+
+RADIXDLT_JAVA_CLIENT_LIBRARY_NAME_OF_REPO_AND_ASSUMED_NAME_OF_DIRECTORY='radixdlt-java'
+RADIXDLT_CORE_NAME_OF_REPO_AND_ASSUMED_NAME_OF_DIRECTORY='radixdlt-core'
+RADIXDLT_ENGINE_NAME_OF_REPO_AND_ASSUMED_NAME_OF_DIRECTORY='radix-engine-library'
+# co-co-co-combo breaker!
+RADIXDLT_REGRESSION_TESTS_NAME_OF_REPO_AND_ASSUMED_NAME_OF_DIRECTORY='RadixRegressionTests' 
+
+# Arguments:
+# 	$1 = git_repo_absolute_path: String
+# 	$2 = git_command: String
+function git_command() {
+	local git_repo_absolute_path="$1"
+	local git_command="$2"
+	echo $(git --git-dir="$git_repo_absolute_path" $git_command)
+}
+
+# Arguments:
+# 	$1 = git_repo_absolute_path: String
+# 	$1 = git_rev_parse_argument: String
+function git_rev_parse_head() {
+	local git_repo_absolute_path="$1"
+	local git_rev_parse_argument="$2"
+	echo $(git_command "$git_repo_absolute_path" "rev-parse $git_rev_parse_argument HEAD")
+}
+
+# Arguments:
+# 	$1 = git_repo_absolute_path: String
+function git_branch_name() {
+	local git_repo_absolute_path="$1"
+	echo $(git_rev_parse_head "$git_repo_absolute_path" --abbrev-ref)
+}
+
+# Arguments:
+# 	$1 = git_repo_absolute_path: String
+function git_last_commit_hash() {
+	local git_repo_absolute_path="$1"
+	echo $(git_rev_parse_head "$git_repo_absolute_path" --short)
+}
+
+# Arguments:
+# 	$1 = git_repo_absolute_path: String
+function git_summary_of_repo() {
+	local git_repo_absolute_path="$1"
+	if test -d "$git_repo_absolute_path"; then
+		local branch_name=$(git_branch_name "$git_repo_absolute_path")
+		local commit_hash=$(git_last_commit_hash "$git_repo_absolute_path")
+		echo "#: $commit_hash, ᛋ: '$branch_name'"
+	else
+		echo "$ERROR_MSG_NO_SUCH_FOLDER"
+	fi
+}
+
+
+echo "⚠️ This script will only work if it is placed in the repo 'radixdlt-java-common', and assuming all other Radix Java repos share the same root as that repo. All repos MUST also be placed in a folder with the exact same name of the repo. ⚠️"
```

Comment:
This line seems a bit excessively long for a console script.  Can we try to limit to 80 chars?